src/Database/DBStructure.php

```diff
@@ -1818,6 +1818,20 @@ public static function definition() {
 						"PRIMARY" => ["uid", "iid"],
 						]
 				];
+		$database["verify"] = [
+				"comment" => "Store token to verify contacts",
```

Comment:
Can you be more precise? Which contacts are we verifying?

Reply:
Mhhh this is a little bit problematic. This table can be used to verify more than just contacts. At the moment we use it only for contact verification.
Do you have any suggestions?

Reply:
`openwebauth_token`?